ansible-scylla-node/tasks/adjust_keyspace_replication.yml

```diff
@@ -24,9 +24,23 @@
   loop: "{{ groups['scylla'] }}"
   run_once: true
 
+- set_fact:
+    scylla_admin_username: "{{ scylla_admin_default_user }}"
+    scylla_admin_password: "{{ lookup('ini', scylla_admin_default_user, section='cql_credentials', allow_no_value=true, default=omit, file=inventory_file) }}"
+  when: "scylla_admin_default_user is defined and 'cql_credentials' in groups"
+
+- fail:
+    msg: "Make sure that {{ scylla_admin_default_user }} is defined in the 'cql_credentials' section"
+  when: "scylla_admin_default_user is defined and scylla_admin_password is not defined"
+
+- set_fact:
+    scylla_admin_username: "cassandra"
+    scylla_admin_password: "cassandra"
+  when: scylla_admin_username is not defined or scylla_admin_password is not defined
```

Comment:
Here you don't do that and I like it more this way ;)

Reply:
How can it be that `scylla_admin_username` is defined but `scylla_admin_password` is not defined here? You were supposed to fail on that before, no?

Wouldn't conditioning on `scylla_admin_default_user` instead be enough?

Reply:
Yes, it's enough. Fixed.